app/jobs/patient_list_download_job.rb

```diff
@@ -0,0 +1,21 @@
+class PatientListDownloadJob < ApplicationJob
+  queue_as :default
+  self.queue_adapter = :sidekiq
+
+  def perform(recipient_email, model_type, params)
+    case model_type
+    when 'district' then
+      district_name = params[:district_name]
+      organization  = Organization.find(params[:organization_id])
+      model = OrganizationDistrict.new(district_name, organization)
+      model_name = district_name
```

Comment:
Can this be named something else? Got thrown off thinking that `model_name` was the name of the model itself like `Facility` or `District`. Don't really have a suggestion though.

Reply:
You're right, this is all super confusing. Can I fix in a separate PR? I'd like to separate district from facility concerns, and just make them use a single partial, instead of having these conditionals all over.